net/net/src/TMessage.cxx

```diff
@@ -82,12 +82,18 @@ TMessage::TMessage(void *buf, Int_t bufsize) : TBufferFile(TBuffer::kRead, bufsi
    fInfos      = nullptr;
    fEvolution  = kFALSE;
 
+   // The receiving socket owns the passed buffer
+   ResetBit(kIsOwner);
```

Comment:
This does not sounds consistent with the `TBufferFile` behavior.  We may need to extend the interface to match.
The corresponding (and called via the base class) constructor:
```
TBufferFile(TBuffer::EMode mode, Int_t bufsiz, void *buf, Bool_t adopt = kTRUE, ReAllocCharFun_t reallocfunc = nullptr);
```
where the default is to adopt/take-ownership of the passed buffer.

Reply:
Moved to https://github.com/root-project/root/pull/17628